### ISSUE TITLE
Bypassing expulsion pylons 

### ISSUE THREAD

**Issue by TheDyreBanana** [jeremiahwinsley/pylons#6]
I just realised the issue fixed in issue #2 actually makes it so now you can just bypass people's pylons with a bed and just setting your spawn inside that chunk. I think it might be possible to use a cmd to set the player spawn inside it too if you lag the server and execute the cmd at the right moment but Idk I don't really have a way to test this atm.

**Comment by jeremiahwinsley** [comment]
Yes, this has always been possible, but I'm not sure there's a way to fix it without turning this into an actual claiming mod. It was meant to be used alongside FTB Chunks, where claiming would prevent placing the bed. The `/spawnpoint` command requires OP, so that's not really a consideration anyways.

**Comment by TheDyreBanana** [comment]
Ah okay, I figured this was the case but I was curious if there was a way to maybe check if a player is setting their spawn inside the expulsion zone and then deny it if they aren't the owner. I imagine this would probably have problems though. I will admit it would be interesting if this became a sort of light weight slim version of claiming that was just able to update faster and be less feature rich (this can also be a benefit for some pack devs or servers where they want alternatives to ftbchunks). However I know that's probably way out of the project scope and a big shift in terms of direction lol.
